business-registry-dashboard/app/components/Modal/Base.vue

```diff
@@ -17,6 +19,13 @@ defineProps<{
 defineEmits<{
   modalClosed: [void]
 }>()
+
+// Remove focus from close button after modal mount to prevent it from being auto-focused
+// This ensures the close button isn't immediately highlighted when the modal opens
+onMounted(async () => {
+  await nextTick()
+  closeButtonRef.value?.$el?.blur()
+})
```

Comment:
Would it be possible to focus the primary action button (or first entry field) instead?

By the way, mounted is only called when the component is rendered (eg, using if). It doesn't get called if you hide and then show this component. I didn't check how this component is rendered but FYI.

Reply:
Sev, so I played around with this a bit. What you said makes sense but I believe the UX team want nothing focused at first.

I tried for example focusing the card, this is what it looks like which is ugly:
![card autofocus](https://github.com/user-attachments/assets/34125fef-28df-4c76-b438-e5bc7a9e10d7)

I tried also focusing the primary content, this is what it looks like:
![content focused](https://github.com/user-attachments/assets/29da12f8-4c21-4c0a-bf98-4fb0c7ad1fe8)

Also, focusing the primary action, same thing. Let me know if you're OK with this or would like me to find another solution.

Reply:
Focusing the primary action looks bad?

Anyway, if UX team wants nothing focused, so be it. (Sometimes I push back on them a bit but not this time, based on your experiments.)

I agree to resolve this conversation.